packages/gatsby-link/src/rewrite-link-path.js

```diff
@@ -15,7 +15,16 @@ function absolutify(path, current) {
   if (isAbsolutePath(path)) {
     return path
   }
-  return resolve(path, current)
+
+  const option = getGlobalTrailingSlash()
+
+  const absolutePath = resolve(path, current)
```

Comment:
```suggestion
  const option = getGlobalTrailingSlash()
  const absolutePath = resolve(path, current)
```

I'd say this is a nit and a code style question, but I personally would group it together. I use empty lines as separators of blocks of logic or functionality.

In this case the "block of functionality" is setting variables to be used later, so I'd group them together. Then there's an empty line and the next block of logic follows with the `if` statement

Reply:
Sure! No strong take on this on really 😉